examples/alphaTestExamples/pruneRetrieveError.md

```diff
@@ -0,0 +1,201 @@
+[kind]: https://github.com/kubernetes-sigs/kind
```

Comment:
I don't think we should keep adding tests using this framework. The mdrip framework is useful for providing executable examples, but I don't think these are being used for that.

While the Ginkgo tests aren't perfect, they are definitely better than using the mdrip framework.